build/configs/rtl8721csm/security_hal_test_tz/defconfig

```diff
@@ -0,0 +1,1276 @@
+#
```

Comment:
what is the base defconfig for this? is it taken from rdp or hello? 
Is it copied from .config after it got generated?

Reply:
it is copied from .config after it got generated